platform-tests/src/test/kotlin/org/eclipse/deeplearning4j/frameworkimport/frameworkimport/onnx/importer/TestOnnxFrameworkImporter.kt

```diff
@@ -1,17 +1,84 @@
 package org.eclipse.deeplearning4j.frameworkimport.frameworkimport.onnx.importer
 
 import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Disabled
 import org.junit.jupiter.api.Tag
 import org.junit.jupiter.api.Test
+import org.nd4j.autodiff.samediff.serde.SDZSerializer
+import org.nd4j.common.config.ND4JSystemProperties
 import org.nd4j.common.io.ClassPathResource
 import org.nd4j.common.tests.tags.TagNames
 import org.nd4j.linalg.api.buffer.DataType
+import org.nd4j.linalg.api.ndarray.INDArray
 import org.nd4j.linalg.factory.Nd4j
 import org.nd4j.onnxruntime.runner.OnnxRuntimeRunner
 import org.nd4j.samediff.frameworkimport.onnx.importer.OnnxFrameworkImporter
+import java.io.BufferedInputStream
+import java.io.File
+import java.io.FileOutputStream
+import java.net.HttpURLConnection
+import java.net.SocketTimeoutException
+import java.net.URL
+import java.nio.channels.Channels
+import java.util.*
 
 @Tag(TagNames.ONNX)
 class TestOnnxFrameworkImporter {
+    @Test
+    fun testOther2() {
+        Nd4j.getEnvironment().isFuncTracePrintJavaOnly = true
+        val importer = OnnxFrameworkImporter()
+        val filePath = "/home/agibsonccc/Documents/GitHub/deeplearning4j/bge-base-en-v1.5-optimized.onnx";
```

Comment:
hardcoded file path to local file.

Reply:
Ah this was accidentally added. Thanks!@